packages/ws-server/src/webSocketServer.ts

```diff
@@ -99,7 +103,11 @@ app.ws.use((ctx) => {
             let subscriptionId;
 
             if (event === 'logs') {
-                subscriptionId = relay.subs()?.subscribe(ctx.websocket, event, filters);
+                if(!getMultipleAddressesEnabled() && Array.isArray(filters.address) && filters.address.length > 1) {
+                    response = predefined.INVALID_PARAMETER(4, 'Only one address is allowed');
```

Comment:
What's the `4`?

Reply:
Same question, also avoid using magic numbers if possible.

Reply:
According to the Predefined and existing error `INVALID_PARAMETER` the first parameter is the index of the parameter that is `INVALID`, and given this example of the input, seems to me to be the `4` parameter.
```
{
    "jsonrpc":"2.0",
    "id": 2, 
    "method": "eth_subscribe", 
    "params": ["logs",
        {
        "address": ["0x6721d86A8405C8683dA3CAEBCbeb4c690EdAE963","0xb3611238Ecda2CB930d1C1A8d065743bfCbf7452"]
        }
    ]    
}
```

But now that I think about it, is a JSON, and as such, it could have the same parameters in different order and still should work... (also, if we were to count `0`, it would be the third parameter...
I will look up for the index of the "params" parameter and return that instead of the hardcoded `4`

Reply:
Changed my mind about the above comment, when realized that then Predefined error `INVALID_PARAMETER` also supports an string instead of a number for the "index" of the parameter that is invalid so, changed it to something more descriptive and leaved a comment on the code explaining it. `params[1].address`

                    // [1] is the second object in the expected array of parameters that contains the filters (address and topics) for the subscription